build.cake

```diff
@@ -158,41 +144,29 @@ Task("UpdateAssemblyInfo")
 // BUILD ENGINE
 //////////////////////////////////////////////////////////////////////
 
+MSBuildSettings CreateMSBuildSettings(string target) => new MSBuildSettings()
+    .SetConfiguration(configuration)
+    .SetVerbosity(Verbosity.Minimal)
+    .WithProperty("PackageVersion", productVersion)
+    .WithTarget(target);
```

Comment:
ℹ This helper method makes it easy to drop in workarounds like `.WithProperty("AddSyntheticProjectReferencesForSolutionDependencies", "false")` in a single place, which I need right now to work around `https://github.com/Microsoft/msbuild/issues/3626`.